client/components/App.js

```diff
@@ -6,12 +6,22 @@ import "../assets/scss/main.scss"
 
 import { Route, Switch, BrowserRouter } from "react-router-dom"
 
+import Index from "./Index"
+import PetTypeIndex from "./PetTypeIndex"
+
 const App = props => {
   useEffect(() => {
     $(document).foundation()
   }, [])
 
-  return <div className="callout primary">Replace this div with your Router.</div>
+  return (
+    <BrowserRouter>
+      <Switch>
+        <Route exact path="/" component={Index} />
+        <Route exact path="/pets" component={PetTypeIndex} />
+      </Switch>
```

Comment:
We can actually remove line 9 and 20 and handle the redirect on our express server.  This will help avoid needing to create a component who's only purpose is to redirect